packages/sdk/src/conversation/interface.ts

```diff
@@ -313,3 +416,31 @@ export interface ConversationOptions {
     enabled?: boolean;
   };
 }
+
+/**
+ * Interface for user to customize sso execution activity handler
+ */
+export interface BotSsoExecutionActivityHandler {
```

Comment:
Can we add API document for each APIs to provide some guidance on implementing a customized SSO activity handler？